mrp_product_use_case/README.rst

```diff
@@ -0,0 +1,72 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+====================
+MRP Product Use Case
+====================
+
+This module add MRP Product Use Case functionality
```

Comment:
You should explain here what is "the product use case functionality". For instance "Allows you to find the product usages across all Bill of Materials".